crates/cairo_client/cairo/src/lib.cairo

```diff
@@ -102,21 +101,22 @@ fn verify_interface(
     let origin = data.arr_slice(client_data_json_len + challenge_len, origin_len);
     let auth_index = client_data_json_len + challenge_len + origin_len;
     let authenticator_data = data.arr_slice(auth_index, data.len() - auth_index);
-    match verify(
-        pub_p,
-        r,
-        s,
-        type_offset,
-        challenge_offset,
-        origin_offset,
-        client_data_json,
-        challenge,
-        origin,
-        authenticator_data
-    ) {
-        Result::Ok => 0,
-        Result::Err(_) => 1
-    }
+    // match verify(
```

Comment:
are we planning to add this back later?

Reply:
My idea is to entirely remove `cairo_client` because we have similar functionality developed in [Cairo Args Runner](https://github.com/neotheprogramist/cairo-args-runner).
I would opt to use it and the approach we've discussed recently about testing and bring it back in the testing task.

Reply:
To clarify: I've commented out this part of the code, because during recent changes the signature of the verify method changed and fixing it here would require some work. Cairo Args Runner has the same functionality as provided by the cairo_client. In the next PR the entire cairo_client folder is removed and we are planning on bringing it back as a dependency during test migration.

Reply:
cool makes sense. lgtm!